packages/app/src/components/contract/VerificationButton.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <Button
+    class="contract-verification-link"
+    :data-testid="$testId.contractVerificationButton"
```

Comment:
Not sure if it should be a part of the component (or how it's used in general).

Reply:
yep, it's for e2e